yin_yang/__main__.py

```diff
@@ -12,6 +12,7 @@
 from PySide6.QtWidgets import QSystemTrayIcon, QMenu
 from systemd import journal
 
+sys.path.append(str(Path(__file__).parent.parent))
```

Comment:
What is your intention here? A comment would be helpful

Reply:
Import local model in VScode need add project location to path and I forgot to revert it